src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-classes": {
```

Comment:
I opened a bug about the untransformed class + transformed spread problem (https://phabricator.babeljs.io/T7467), but no solution appeared so far.

I propose throwing a compilation error asking for `transform-es2015-classes` to be enabled if it ever encounters a `super` with spread arguments, but that's something the spread plugin has to do.